src/integrationTest/java/uk/gov/hmcts/reform/pip/data/management/service/ValidationServiceTest.java

```diff
@@ -479,28 +479,28 @@ private static Stream<Arguments> nonStrategicParameters() {
                          "data/non-strategic/technology-and-construction-court-kb-daily-cause-list/"
                              + "technologyAndConstructionCourtKbDailyCauseList.json"),
             Arguments.of(ListType.ADMIRALTY_COURT_KB_DAILY_CAUSE_LIST,
-                         "data/non-strategic/admiralty_court_kb_daily_cause_list/"
+                         "data/non-strategic/admiralty-court-kb-daily-cause_list/"
```

Comment:
There's an underscore remaining in this one

Reply:
fixed.